Exercicios/Exercicio6.sql

```diff
@@ -0,0 +1,5 @@
+--6. LISTE OS PROFESSORES CUJO O M�S DE ADMISS�O SEJA Novembro OU Dezembro.
+SELECT * INTO Nov_Dec_Teacher FROM [Teacher]
+WHERE month(AdmitionDate) BETWEEN '11' AND '12';
```

Comment:
Mudar para cláusula 'IN' e o valores não são strings.

Reply:
Nesse caso, não pode usar o BETWEEN por que? Eu pensei que, como são datas, poderia usar para dizer que estão entre novembro e dezembro.